central/credentialexpiry/service/service_impl.go

```diff
@@ -85,23 +92,19 @@ func ensureTLSAndReturnAddr(endpoint string) (string, error) {
 	return fmt.Sprintf("%s:443", server), nil
 }
 
-func (s *serviceImpl) maybeGetExpiryFomScannerAt(ctx context.Context, endpoint string) (*types.Timestamp, error) {
-	addr, err := ensureTLSAndReturnAddr(endpoint)
```

Comment:
would some form of this be useful for Scanner v4, too?

Reply:
Assuming referring to `ensureTLSAndReturnAddr`?

If so then no I don't believe so.  `ensureTLSAndReturnAddr` ensures the endpoint starts with `https://` and then attempts to identify the 'domain+port' - which is not applicable to `gRPC` endpoints.